blocks/graph.py

```diff
@@ -209,6 +209,13 @@ def replace(self, replacements):
                 replacement_keys_cur.append(input_)
                 replacement_vals_cur.append(replacements[input_])
 
+        # Add outputs of the computation graph
+        for output in self.outputs:
+            if output not in replacements:
```

Comment:
Is it a bad practice to use `continue`? I tried to decrease the number of indents.